src/test/configs/dockerfile-with-target/.devcontainer.json

```diff
@@ -8,7 +8,14 @@
 		"options": [ "--label", "test_build_options=success" ]
 	},
 	"postCreateCommand": "touch /tmp/postCreateCommand.testmarker",
-	"postStartCommand": "touch /tmp/postStartCommand.testmarker",
```

Comment:
Shouldn't we still keep this `"touch /tmp/postStartCommand.testmarker` ?

Reply:
reverted